pkg/executables/kubectl.go

```diff
@@ -1311,3 +1311,27 @@ func (k *Kubectl) GetDaemonSet(ctx context.Context, name, namespace, kubeconfig
 
 	return obj, nil
 }
+
+func (k *Kubectl) GetPackagesFromKubectl(ctx context.Context, packageInstanceName, kubeconfig string, output string) (string, error) {
+	params := []string{
+		"get", "packages", kubeconfigFlag, kubeconfig, namespaceFlag, constants.EksaPackagesName,
```

Comment:
nit: I understand the desire to dry out hardcoded strings but I think that doing this for the flags hurts a little bit the readability. And I hardly doubt that those flags are ever going to change.